superset/utils/screenshots.py

```diff
@@ -102,6 +102,8 @@ def __init__(
         self._window: WindowSize = window or (800, 600)
         config_auth_func = current_app.config.get("WEBDRIVER_AUTH_FUNC", auth_driver)
         self._auth_func = auth_func or config_auth_func
+        self._element_locate_wait = current_app.config.get("SCREENSHOT_LOCATE_WAIT", 10)
```

Comment:
add SCREENSHOT_LOCATE_WAIT to the config and set default there rather than in code.
here use:
```
 current_app.config["SCREENSHOT_LOCATE_WAIT"]
```